capstone/src/main/java/com/google/univiz/api/SearchResourceImpl.java

```diff
@@ -0,0 +1,19 @@
+package com.google.univiz.api;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Implementation of SearchResource Interface */
+public class SearchResourceImpl implements SearchResource {
+
+  public List<SearchData> getSearchSuggestions(List<String> collegeNames) {
+    List<SearchData> searchDataList = new ArrayList<>();
+    List<SuggestionData> collegeSuggestions = getSuggestions(collegeNames);
```

Comment:
What is `getSuggestions`?

Reply:
this would be a function obtained from a function in SuggestionAPI, which communicates with the REST API. Should I add a comment here?

Reply:
So for you to be able to use this method, first you have to add the SuggestionAPI as a field in this class. But that API is not yet in master, so first you have to wait for the corresponding PR to be merged. All you can do right now is to add a TODO.